docs/spot-instances.md

```diff
@@ -12,7 +12,7 @@ In the following examples at least 1 worker group that uses on-demand instances
 
 ```yaml
 nodeSelector:
-  kubernetes.io/lifecycle: normal
+  node.kubernetes.io/lifecycle: normal
```

Comment:
@gertjangaillet mentioned in the comment here (https://github.com/terraform-aws-modules/terraform-aws-eks/issues/856#issuecomment-623155710) that if you apply the old labels on 1.16, the nodes don't actually come up because the `kubelet` process rejects them.

Are you saying you want the **1.15** labels, but commented out?

Reply:
Just a note saying something like: for eks 1.15 and 1.14 you have to use `kubernetes.io/lifecycle`.

Reply:
Done.

Reply:
The new labels will work on versions older than 1.16. I chose `kubernetes.io/lifecycle=spot` for the docs but it can be any key/value. It's only to help with scheduling, like with a node selector. So perhaps we don't need the note at all?

Reply:
@max-rocket-internet Good point. Should I revert it? @barryib

Reply:
good point. @sc250024 yes please. Sorry for the extra commit.

Reply:
Yeah I think remove the comments about previous versions to just keep it clean 🙂

Reply:
@max-rocket-internet @barryib Resolved.